### DISCUSSION TITLE
Confused regarding "Standard install has BOSE updates REMOVED"

### DISCUSSION THREAD

**Discussion by googleg** [harman-f/MHI2_US_POG11_K5186_1-MU1476-AIO#7] [Q&A]
Hi folks,
I am trying to fully uderstand the way this rocess works and the ention "Standard install has BOSE updates REMOVED" puzzles me since I believe the process installed a patched FW and not an update per say.
I am about to patch the FW of my Macan which has a Bose sound system so I want to be 100% sure I do not loose sound in the process.

**Comment by stevenpink** [comment]
If you apply the full MIB FW update on a car with Bose stereo, it can cause an issue where you have no sound output until you redo the Bose amp parameterization with PIWIS. This "AIO" (All in One) update version has been configured to skip the update that causes the Bose issues. I've installed this on my car with Bose and it works perfectly.
